docs/src/3.1.Worldview.mdx

```diff
@@ -2,20 +2,20 @@
-| hitmapOnMouseMove    | `boolean`                         | false                | if enabled, the clicked objectId will be returned from the mouse event handler                          |
-| children?            | `React.Node`                      |                      | regl components to render inside the Worldview                                                          |
-| showDebug?           | `?boolean`                        |                      | show debug information                                                                                  |
-| onDoubleClick?       | `MouseHandler`                    |                      |                                                                                                         |
-| onMouseDown?         | `MouseHandler`                    |                      |                                                                                                         |
-| onMouseUp?           | `MouseHandler`                    |                      |                                                                                                         |
-| onMouseMove?         | `MouseHandler`                    |                      |                                                                                                         |
-| onClick?             | `MouseHandler`                    |                      |                                                                                                         |
+| Name                  | Optional | Type                     | Default                | Description                                                                                             |
+| --------------------- | -------- | ------------------------ | ---------------------- | ------------------------------------------------------------------------------------------------------- |
+| `cameraState`         | yes      | `$Shape<CameraState>`    |                        | control the [cameraState](/api/camera), must use with `onCameraStateChange`                             |
+| `onCameraStateChange` | yes      | `?(CameraState) => void` |                        | callback whenever the camera state changes                                                              |
+| `defaultCameraState`  |          | `$Shape<CameraState>`    | `DEFAULT_CAMERA_STATE` | pass in this prop to turn Worldview to an uncontrolled component. `onCameraStateChange` will be ignored |
+| `backgroundColor`     |          | `Vec4`                   | `[0,0,0,1]`            | change the scene background color                                                                       |
+| `hitmapOnMouseMove`   |          | `boolean`                | `false`                | if enabled, the clicked objectId will be returned from the mouse event handler                          |
```

Comment:
All these three are also optional right? (After all, they have defaults.)

Reply:
Maybe we should just note that all fields are optional?

Reply:
Good idea. Will remove the column and add the note instead.